cpu/efm32/include/periph_cpu.h

```diff
@@ -356,28 +357,23 @@ typedef struct {
     timer_dev_t prescaler;  /**< the lower neighboring timer (not initialized for LETIMER) */
     timer_dev_t timer;      /**< the higher numbered timer */
     IRQn_Type irq;          /**< number of the higher timer IRQ channel */
+    uint8_t ch_numof;       /**< number of channels per timer */
 } timer_conf_t;
 /** @} */
 
 
 /**
- * @brief   The implementation can use one LETIMER or two regular timers cascaded
+ * @brief   Use LETIMER as the base timer for XTIMER
  */
-#ifndef EFM32_USE_LETIMER
-#define EFM32_USE_LETIMER   0
+#ifndef CONFIG_EFM32_XTIMER_USE_LETIMER
+#define CONFIG_EFM32_XTIMER_USE_LETIMER   0
```

Comment:
The equivalent Kconfig symbol is not defined.

Reply:
This can be added in follow-up if you prefer.

Reply:
See https://github.com/RIOT-OS/RIOT/pull/14848